events/wc_dark_cult_events.txt

```diff
@@ -1037,62 +1068,92 @@ province_event = {
 		}
 	}
 }
-# The Ruins of Ahn'Qiraj
+# The Ruins of Ahn'Qiraj and Darkness in Uldaur
 narrative_event = {
 	id = WCDAC.5040
 	title = EVTTITLE_WCDAC_5030
-	desc = EVTDESC_WCDAC_5040
-	picture = GFX_evt_ahnqiraj
+	desc = {
+		trigger = {
+			has_global_flag = twilights_hammer_kalimdor
+		}
+		text = EVTDESC_WCDAC_5040_kalimdor
+		picture = GFX_evt_ahnqiraj
+	}
+	desc = {
+		trigger = {
+			NOT = { has_global_flag = twilights_hammer_kalimdor }
+		}
+		text = EVTDESC_WCDAC_5040_northrend
+		picture = GFX_evt_yogg_saron
+	}
 	border = GFX_event_narrative_frame_intrigue
 
 	is_triggered_only = yes
 
 	immediate = {
 		# Used in scripted effects
 		save_event_target_as = target_settler
-		
+
 		religion = old_gods_worship
 		clear_event_target = yes
-		
-		remove_trait = travelling
-		
-		c_hivezora = {
-			# event_target:target_settler and its people settle here
-			settle_as_target_settler_effect = yes
+		any_courtier = {
+			limit = {
+				culture = twilights_hammer
+				can_switch_religion_character_trigger = yes
+			}
+			religion = old_gods_worship
 		}
-		
-		# If you're not independent ruler, gives old titles to liege
-		give_old_titles_to_heir_effect = yes
-		
-		d_crystalvale = {
-			any_direct_de_jure_vassal_title = {
+
+		remove_trait = travelling
+
+		if = {
+			limit = { has_global_flag = twilights_hammer_kalimdor }
+			c_southmoon = {
 				# event_target:target_settler and its people settle here
 				settle_as_target_settler_effect = yes
 			}
+
+			# If you're not independent ruler, gives old titles to liege
+			give_old_titles_to_heir_effect = yes
+
+			c_southmoon = {
+				# Declares war against player
+				target_settler_declare_war_effect = yes
+			}
 		}
-		d_scarab_wall = {
-			any_direct_de_jure_vassal_title = {
+		else = {
+			c_strand_of_ancients = {
 				# event_target:target_settler and its people settle here
 				settle_as_target_settler_effect = yes
 			}
+
+			# If you're not independent ruler, gives old titles to liege
+			give_old_titles_to_heir_effect = yes
+			
+			c_strand_of_ancients = {
+				# Declares war against player
+				target_settler_declare_war_effect = yes
+			}
 		}
-		
-		c_hivezora = {
-			# Declares war against player
-			target_settler_declare_war_effect = yes
-		}
-		
 		clear_settler_flag_effect = yes
-		
-		# Spawns army for start
-		spawn_colonial_troops_effect = yes
 	}
 
-	# I'll spread its will across Kalimdor!
+	# I'll spread its will across Kalimdor/Northrend!
 	option = {
 		name = EVTOPTA_WCDAC_5040
 
-		add_artifact = hammer_of_the_twilight
+		if = {
+			limit = { potential_shadow_priest_wo_training_trigger = yes }
+			hidden_effect = {
+				add_character_modifier = { name = training_self_shadow_priest duration = -1 }
+				recount_experience_effect = yes
+				narrative_event = { id = WCCLS.1300 }
+			}
+		}
```

Comment:
What if character has no magical class?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/80985811-63afcd80-8e38-11ea-9938-08b332abd636.png)

</details>